main.go

```diff
@@ -142,14 +141,16 @@ func main() {
 
 	engine.PrintSettings(&engine.Bot.Settings)
 	if err = engine.Bot.Start(); err != nil {
-		gctlog.Errorf(gctlog.Global, "Unable to start bot engine. Error: %s\n", err)
-		os.Exit(1)
+		errClose := gctlog.CloseLogger()
+		if errClose != nil {
+			log.Fatalf("Unable to close logger. Error: %s\n", errClose)
```

Comment:
Don't make this fatal, that way you can get both errors for logger and gct failure